React: update 'render' and 'component' descriptions (#5676)
DevExpress/devextreme-documentation: concepts/50 React Components/40 Component Configuration Syntax/55 Markup Customization/3 Using a Rendering Function.md

> The following example customizes a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component to display a combination of grid's data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value within cells. The code wraps a rendering function within a [useCallback](https://react.dev/reference/react/useCallback) hook. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.

Reviewer: ```suggestion
The following example customizes a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component to display a combination of grid data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value within cells. The code wraps a rendering function within a [useCallback](https://react.dev/reference/react/useCallback) hook. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.
```

> The following example demonstrates a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component which cells display a combination of grid's data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value. For this task, a rendering function wrapped in [useCallback](https://react.dev/reference/react/useCallback) was used. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.

Reviewer 2: The word "which" should actually be "whose" in this context ("components whose cells display ...").

And then I'm trying to keep everything active (remove passive voice). 

```suggestion
The following example customizes a [DataGrid](/Documentation/Guide/UI_Components/DataGrid/Getting_Started_with_DataGrid/) component to display a combination of grid's data and a [TextBox](/Documentation/Guide/UI_Components/TextBox/Getting_Started_with_TextBox/) value within cells. The code wraps a rendering function within a [useCallback](https://react.dev/reference/react/useCallback) hook. This way you can pass the TextBox value to the column's [cellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#cellRender) property.
```

> [note] You can also define the rendering function outside the component when you don't need to pass any dependencies to the *'render'* property. 

Reviewer 2: ```suggestion
[note] You can also define a rendering function outside of a component if you don't need to pass any dependencies to the *'Render'* property. 
```

    <!-- tab: Function component -->
    import React, { useState, useCallback } from 'react';
    import TextBox from 'devextreme-react/text-box';
    import DataGrid, { Column } from 'devextreme-react/data-grid';

    const dataSource = [1, 2, 3, 4, 5];

    function App() {
        const [value, setValue] = useState('.');

        const onTextBoxValueChange = (e) => {
            setValue(e);
        };

>         const cell = useCallback((e) => <div>{e.data + value}</div>, [value]);

Reviewer 3: e name is usually used for even arguments. But a template is not an event handler (in its general meaning). I would expect something like this: 

```
const cellRender = useCallback((cell) => <div>{cell.data + value}</div>, [value]);
````
